### DISCUSSION TITLE
@nrwl:linter:eslint doesn't auto fix errors?

### DISCUSSION THREAD

**Discussion by kdaang** [nrwl/nx#9102] [General]
When running `nx run app:lint --fix` i expect errors to be auto fixed, but it's not. I can see that the linter is detecting the errors, but it's not applying the fixes to which it should according to the [docs](https://nx.dev/linter/eslint#fix). I'm new to the NX ecosystem and I'm using `@nrwl/linter:eslint`, any ideas on what can be the case?

![image](https://user-images.githubusercontent.com/28932125/155464157-9877fa01-678f-4e12-bdc4-c26025cef9b1.png)
 
 ```
  >  NX   Report complete - copy this into the issue template

   Node : 14.17.6
   OS   : linux x64
   yarn : 1.22.17
   
   nx : 13.8.3
   @nrwl/angular : undefined
   @nrwl/cli : 13.8.3
   @nrwl/cypress : 13.8.3
   @nrwl/detox : undefined
   @nrwl/devkit : 13.8.3
   @nrwl/eslint-plugin-nx : 13.8.3
   @nrwl/express : undefined
   @nrwl/jest : 13.8.3
   @nrwl/js : 13.8.3
   @nrwl/linter : 13.8.3
   @nrwl/nest : 13.8.3
   @nrwl/next : 13.8.3
   @nrwl/node : 13.8.3
   @nrwl/nx-cloud : undefined
   @nrwl/react : 13.8.3
   @nrwl/react-native : undefined
   @nrwl/schematics : undefined
   @nrwl/storybook : 13.8.3
   @nrwl/tao : 13.8.3
   @nrwl/web : 13.8.3
   @nrwl/workspace : 13.8.3
   typescript : 4.5.5
   rxjs : 7.5.4
   ---------------------------------------
   Community plugins:

 ```

**Comment by devinrhode2** [comment]
Checkout https://nx.dev/recipes/running-tasks/pass-args-to-commands#pass-args-in-the-targetdefaults-for-the-task
